clients/e-shop-demo/e-shop-frontend/src/data/tooltips.ts

```diff
@@ -33,7 +33,7 @@ export const steps: Step[] = [
 	{
 		title: 'Checkout',
 		target: '.form-select',
-		content: 'Select one of the example credentials to verify your age. In a real-world scenario the credential would be provided by your wallet.',
+		content: 'Select one of the example credentials to verify your age. In a real-world scenario the credential would be provided by your wallet, ie., after issued by a bank',
```

Comment:
. is missing at the end

last sentence is hard to read maybe as following would be better

In a real-world scenario the credential was issued by a trustworthy entity like a bank and would be provided by your credential wallet.